frontend/src/__tests__/cypress/cypress/pages/servingRuntimes.ts

```diff
@@ -101,7 +105,7 @@ class ServingRuntimes {
   }
 
   shouldEnterData() {
-    cy.get('.view-lines.monaco-mouse-cursor-text').type('test');
+    cy.findAllByRole('presentation').find('.view-lines.monaco-mouse-cursor-text').type('test');
```

Comment:
give a data-testid

Reply:
same here

Reply:
Create a new page object for working with the code editor. Please move all related selectors there. Extend `Contextual`.

Modify `DashboardCodeEditor` to accept a  a test ID that gets assigned to the parent `div` since it doesn't seem like PF `CodeEditor` supports `data-testid`.

There's also no reason for qualifying with `cy.findAllByRole('presentation').` This is such a generic selector that it will find many things in the page.